docs/Control Model.md

```diff
@@ -58,21 +54,14 @@ Figure **NCA Control Model** illustrates the NCA class tree, and shows the categ
 
 The categories shown in figure **NCA Control Model** are as follows:
 
-- The **Blocks** subtree contains the definition of **NcBlock** - see [Blocks](Device%20Model.md#Blockspecs-and-Blockspec-IDs).
-
-- The **Workers** subtree contains definitions of classes that may perform some kind of processing or monitoring function. **SignalWorkers** are specific workers which handle media signal processing.
-**SignalWorkers** has two subtrees:
-    - **Actuators** , which affect signals in various ways;
-    - **Sensors**, which monitor signals.
-
-- The **Managers** subtree contains definitions of classes that perform Device housekeeping functions. Each manager class may be instantiated at most once per NCA device. A few of them must be instantiated for NCA conformance - see [Base Device blockspec](Device%20Model.md#Base-Device-blockspec).
-
-Particularly notable Managers are:
-
-1. **NcDeviceManager**. Contains generic product information (model and serial numbers, for example) and holds overall Device status indicators.
+- The **Blocks** subtree contains the definition of [NcBlock](https://specs.amwa.tv/ms-05-02/branches/v1.0-dev/docs/Framework.html#ncblock) - see [Blocks](Device%20Model.md#Blockspecs-and-Blockspec-IDs).
 
-2. **NcClassManager**. Offers class and data type discovery mechanisms.
+- The [Workers](https://specs.amwa.tv/ms-05-02/branches/v1.0-dev/docs/Framework.html#workers) subtree contains definitions of classes that can perform some kind of processing or monitoring function. [SignalWorkers](https://specs.amwa.tv/ms-05-02/branches/v1.0-dev/docs/Framework.html#ncsignalworker) are specific workers which handle media signal processing.
```

Comment:
"some kind of" feels a little imprecise.
Is a "specific worker" different to a "worker"?
Should worker be capitalised?
```suggestion
- The [Workers](https://specs.amwa.tv/ms-05-02/branches/v1.0-dev/docs/Framework.html#workers) subtree contains definitions of classes that perform processing or monitoring functions. [SignalWorkers](https://specs.amwa.tv/ms-05-02/branches/v1.0-dev/docs/Framework.html#ncsignalworker) are Workers which handle media signal processing.
```